src/utils/random-utils.js

```diff
@@ -95,8 +95,8 @@ const randomUtils = () => {
    * @param {number} n Number of words in the name
    * @param {string} join The character(s) used to join each word
    */
-  const getRandomName = (n, join = '_') =>
-    getNumberArray(n)
+  const getRandomName = (num, join = '_') =>
```

Comment:
Maybe 'length'?

Reply:
thanks for highlighting these. makes sense... i changed this one to numWords